00-HTML-CSS-basics/Ex02/css/layout_styles.css

```diff
@@ -0,0 +1,67 @@
+html {
+	font-family:'Margarine', Times, serif;
+	box-sizing: border-box;
+	background-color: #7d42f4;
+	text-align: center;
+}
+h1 {
+	font-size: 40px;
+	color: white;
+}
+div.container {
+	position: absolute;
+	width: 100%;
+	height: 100%;
+}
+div.box {
+	display: inline-block;
+	position: relative;
+	width: 33.3%;
+	margin: 2% 2%;
+	z-index: 0;
+	height: auto;
+}
+div.inner_box {
+	opacity: 0;
+	display: inline-block;
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 90%;
+	height: 90%;
+	z-index: 1;
+	padding: 5%;
+	background-color: black;
+	color: white;
+	text-align: center;
```

Comment:
The `text-align: center` is already set in the `html` tag. Try to avoid additional css